src/ts/fileDescriptorTSServices.ts

```diff
@@ -144,18 +146,14 @@ export function printFileDescriptorJSServices(fileDescriptor: FileDescriptorProt
       const requestMessageTypeName = getFieldType(MESSAGE_TYPE, method.getInputType().slice(1), "", exportMap);
       const responseMessageTypeName = getFieldType(MESSAGE_TYPE, method.getOutputType().slice(1), "", exportMap);
 
-      printer.print(`${service.getName()}.${method.getName()} = `);
-      printJavaScriptClass(printer, {
-        className: method.getName(),
-        statics: {
-          methodName: `"${method.getName()}"`,
-          service: service.getName(),
-          requestStream: method.getClientStreaming(),
-          responseStream: method.getServerStreaming(),
-          requestType: requestMessageTypeName,
-          responseType: responseMessageTypeName,
-        }
-      });
+      printer.printLn(`${service.getName()}.${method.getName()} = {`);
```

Comment:
Service Methods are now exported as an plain old javascript object instead of a 'class'